content/local-setup.md

```diff
@@ -221,9 +222,9 @@ The output of this command will include instructions for a deploying your progra
 
 ```text
 To deploy this program:
-  $ solana program deploy /Users/James/Dev/Work/solana-hello-world-local/target/deploy/solana_hello_world_local.so
```

Comment:
Fine leaving this how you put it, but just a heads up this isn't a code formatting thing. This is something printed to the console so it actually *is* only printed with two spaces rather than a tab.